### PR TITLE
Rework documentation and execution infrastructure

### PR DESCRIPTION
**Pull request by scoopex**
- Add a wrapper script for execution
- Improved documentation
- Add docker build
- Removed a execution warning

### CODE REVIEW

**Comment by fkr** [comment]
@scoopex for the documentation: you change a lot of items to relative paths instead of pointing to the absolute url of the repository. I think the docs bits should be viewed under the "being included into docs.scs.community' aspect. 

@maxwolfs

File: docs/overview.md [hunk 1]
```diff
@@ -10,59 +10,62 @@ The SCS Health Monitor project aims to ensure the robustness and reliability of
 
 To get started with the SCS Health Monitor project, follow these steps:
 
-1. Clone [the repository](https://github.com/SovereignCloudStack/scs-health-monitor) to your local machine.
+1. Clone this repository to your local machine.
```
[the remark this answers is not in the document]
Comment (scoopex) [reply docs/overview.md]: Originally there were two versions of the document, one as README.md on the top level directory and the ‘overview.md’ document. I first revised the ‘README.md’ and later realized that, as it now turns out, there was an almost identical version.  I have now corrected this by adding a linkand i added a refernce to the repo.

**Comment by scoopex** [comment]
> @scoopex for the documentation: you change a lot of items to relative paths instead of pointing to the absolute url of the repository. I think the docs bits should be viewed under the "being included into docs.scs.community' aspect.
> 
> @maxwolfs

This was also caused by the duplicate content alignment.
Fixed that. I will test again after merging that.

File: cloud_level_testing/features/steps/tools.py [hunk 2]
```diff
@@ -813,18 +813,18 @@ def create_wait_script(conn_test, testname):
...
 
-    script_content = f"""
+    script_content = '''
             #!/bin/bash
             let MAXW=100
             if test ! -f /var/lib/cloud/instance/boot-finished; then sleep 5; sync; fi
-            while test \$MAXW -ge 1; do
-            if type -p "{conn_test}">/dev/null || type -p "{secondary}">/dev/null; then exit 0; fi
+            while test $MAXW -ge 1; do
+            if type -p "%s">/dev/null || type -p "%s">/dev/null; then exit 0; fi
```
Comment (piobig2871) [inline cloud_level_testing/features/steps/tools.py:821]: more "pythonic" way to handle formatting is to use the .format() method

Comment (scoopex) [reply cloud_level_testing/features/steps/tools.py:822]: I know, normally i use f-strings or .format, but in this particular situation i use the c-style printf formatting to stop the complain python interpreter :-)
I added textwrap.dedent to beautify the created script.

**Review by bitkeks — APPROVED** [review]
LGTM!

Links in docs can also be changed to the Github permalink, so that path changes in the repo don't break docs links.

**Pull request merged**